scripts/mobile-center-login.sh

```diff
@@ -27,7 +24,7 @@ fi
 
 # Log in to Mobile Center
 echo "Logging in to mobile center..."
-mobile-center login -u "$USERNAME" -p "$PASSWORD"
+mobile-center login --token "$MOBILE_CENTER_API_TOKEN"
```

Comment:
Nope, usually command line params are either -t or --token, not -word (on unix -word woud generally mean -w -o -r -d)